crypto/dilithium/pqcrystals_dilithium_ref_common/packing.c

```diff
@@ -84,15 +90,16 @@ void pack_sk(uint8_t sk[CRYPTO_SECRETKEYBYTES],
     sk[i] = tr[i];
   sk += TRBYTES;
 
-  for(i = 0; i < L; ++i)
-    polyeta_pack(sk + i*POLYETA_PACKEDBYTES, &s1->vec[i]);
-  sk += L*POLYETA_PACKEDBYTES;
+  for(i = 0; i < params->l; ++i)
+    polyeta_pack(params, sk + i * params->poly_eta_packed_bytes, &s1->vec[i]);
+  sk +=  params->l*params->poly_eta_packed_bytes;
 
-  for(i = 0; i < K; ++i)
-    polyeta_pack(sk + i*POLYETA_PACKEDBYTES, &s2->vec[i]);
-  sk += K*POLYETA_PACKEDBYTES;
 
-  for(i = 0; i < K; ++i)
+  for(i = 0; i <  params->k; ++i)
+    polyeta_pack(params,sk + i * params->poly_eta_packed_bytes, &s2->vec[i]);
+  sk +=  params->k*params->poly_eta_packed_bytes;
```

Comment:
```suggestion
  sk +=  params->k * params->poly_eta_packed_bytes;
```

Reply:
Thanks, fixed in [ea859d1](https://github.com/aws/aws-lc/pull/1910/commits/ea859d1b4a527b801a531e415122f51475802817).